searchbar.js

```diff
@@ -4,45 +4,66 @@ var lessonOneContent = lessonOne.textContent.toLowerCase();
 var lessonOneArr= lessonOneContent.split(/\W/g)
 var lessonTwo = document.getElementById("lesson-two");
 var lessonTwoContent = lessonTwo.textContent.toLowerCase();
-var lessonTwoArr= lessonTwoContent.split(/\W/g)
+var lessonTwoArr = lessonTwoContent.split(/\W/g)
+var noMatchesBanner = document.getElementById("noMatchesBanner")
 //use split on the string, instead of checking is equal, check if text input is within the sting (sub-sting) 
 
 function searchFunction(e) {
     var text = e.target.value.toLowerCase();
     let isLessonOneAMatch = false;
     let isLessonTwoAMatch = false;
     let userInput = text.split(/\W/g);
-    console.log(userInput)
-    console.log(lessonOneArr)
-    console.log(lessonTwoArr)
+    // console.log(userInput)
+    // console.log(lessonOneArr)
+    // console.log(lessonTwoArr)
+
    for (let i = 0; i < userInput.length; i++) {
-        
-        for (let j = 0; j < lessonOneArr.length; j++) {
-            if (userInput[i] === lessonOneArr[j]) {
-                isLessonOneAMatch = true;
-            }
-        };
-        for (let k = 0; k < lessonTwoArr.length; k++) {
-            if (userInput[i] === lessonTwoArr[k]) {
-                isLessonTwoAMatch = true;
-            }
-        };
-    };
-    if (isLessonOneAMatch === true && isLessonTwoAMatch === true) {
-        console.log("they are both matches");
-    } else
-        if (isLessonTwoAMatch === true) {
-                console.log("lesson Two is a match");
-    } else
-    if (isLessonOneAMatch === true) {
-        console.log("lesson One is a match");
-    } else {
-            console.log("there are no matches");
+       for (let j = 0; j < lessonOneArr.length; j++) {
+           if (lessonOneArr[j].includes(userInput[i])===true){
+        isLessonOneAMatch =  true
+        // console.log(isLessonOneAMatch)
+    //         if (lessonOneArr.includes(userInput[i]) === lessonOneArr[j]) {
+    //             isLessonOneAMatch = true;
+           }
+         };
+       for (let k = 0; k < lessonTwoArr.length; k++) {
+           if(lessonTwoArr[k].includes(userInput[i])===true){
+        isLessonTwoAMatch =  true
+        // console.log(isLessonTwoAMatch)
+    //         if (userInput[i] === lessonTwoArr[k]) {
+    //             isLessonTwoAMatch = true;
+             }
+         };
+    // };
+       if (isLessonOneAMatch === true && isLessonTwoAMatch === true) {
+           //console.log("they are both matches");
+            lessonOne.style.display = "block";
+           lessonTwo.style.display = "block";
+           noMatchesBanner.style.display = "none";
```

Comment:
When you get into using react, you'll start to be able to set classes on elements dynamically, so it becomes a lot cleaner to do this stuff.